.github/workflows/ci.yml

```diff
@@ -19,4 +19,9 @@ concurrency:
 jobs:
   puppet:
     name: Puppet
-    uses: voxpupuli/gha-puppet/.github/workflows/basic.yml@v3
+    uses: voxpupuli/gha-puppet/.github/workflows/beaker.yml@icinga
```

Comment:
before we merge https://github.com/voxpupuli/puppet-icingadb/pull/51/files, we need to merge the gha-puppet changes back to v3 branch.

Reply:
https://github.com/voxpupuli/gha-puppet/pull/73